web/Web/web/search/templates/search/search.html

```diff
@@ -51,124 +58,147 @@
 {% include 'modals/document_modal.html' %}
 <!-- Judging criteria modal -->
 {% include 'modals/additional_judging_criteria_modal.html' %}
-<!-- Discovery failed to receive judgment toast -->
-{% include 'toasts/discovery_failed_to_receive_judgment.html' %}
```

Comment:
Why is this removed?

Reply:
This is not removed actually, the file reformatting has led to these kinds of false removed code alerts. I will fix it so you can check them more easily.